pkg/api/config.go

```diff
@@ -56,6 +56,8 @@ type Config struct {
 
 	// geneva logging sector
 	GenevaLoggingSector string `json:"genevaLoggingSector,omitempty"`
+	GenevaMDMAccount    string `json:"genevaMDMAccount,omitempty"`
```

Comment:
We could use docs about these new fields.

Reply:
docs.go?

Reply:
godoc, comments on top of the fields

Reply:
@mjudeikis bump